infrastructure/staging/main.tf

```diff
@@ -0,0 +1,250 @@
+module "goobi" {
```

Comment:
It would be more pleasing if there could a single module and two instantiations (one for each environment). Though I think I remember someone saying that wasn't an option as the two envs were to be different - again it would be nice if they were as similar as possible (as expressed through a single module with perhaps options for scaling services as appropriate for an env).

Reply:
As mentioned above, we will get rid of the shellservers some time in the future. After that, we will have the single module with two instantiations